### PR TITLE
Readme and chain option for merge repo

### PR DESCRIPTION
**Pull request by tstrachota**
I'm adding readme and `:chain` option to make the interface closer to `Chain` repo.

### CODE REVIEW

File: lib/fast_gettext/translation_repository/merge.rb [hunk 1]
```diff
@@ -10,6 +10,9 @@ class Merge < Base
       def initialize(name, options={})
         clear
         super(name, options)
+        options.fetch(:chain, []).each do |repo|
```
Comment (grosser) [inline lib/fast_gettext/translation_repository/merge.rb]: does it make sense to ever have an empty chain ?

File: Readme.md [hunk 2]
```diff
@@ -170,6 +170,22 @@ the first cannot translate a given key, the next is asked and so forth.
...
+###Merge
+In some cases you can benefit from using merge repositories as an alternative to chains. They behave nearly the same. The difference is in the internal
+data structure. While chain repos iterate over the whole chain for each translation, merge repositories select and store the first translation at the time
+a subordinate repository is added. This puts the burden on the load phase and speeds up the translations.
+
+    repos = [
+      FastGettext::TranslationRepository.build('new', :path=>'....'),
+      FastGettext::TranslationRepository.build('old', :path=>'....')
+    ]
+    domain = FastGettext.add_text_domain 'combined', :type=>:merge, :chain=>repos
```
Comment (grosser) [inline Readme.md]: 1.9 syntax ? :)
`type: :merge, chain: repos`

Comment (tstrachota) [inline Readme.md]: Hehe, again :)

File: lib/fast_gettext/translation_repository/merge.rb (hunk 1 above)
Comment (tstrachota) [inline lib/fast_gettext/translation_repository/merge.rb]: I think it does. You can create an empty repo and use `add_repo` later.
Also empty chain is still a chain and IMO it's usually simpler to accept empty input than protecting each usage with a condition.

**Comment by tstrachota** [comment]
I changed hashes the whole readme to 1.9 syntax for consistency's sake.

@grosser once we merge this, may I ask you to release new version?

**Comment by grosser** [comment]
1.2.0 is out!

**Comment by tstrachota** [comment]
Thank you, @grosser !

**Pull request merged**